spec/peep_spec.rb

```diff
@@ -0,0 +1,24 @@
+
+describe Peep do
+
+  context 'Demonstration of how datamapper works' do
+
+    before(:each) do
+    @user = User.create(:name => "Anna Klimas",
+              :username => "ania2ab",
+              :email => "anna@klimas.com",
+              :password => "apple",
+              :password_confirmation => 'apple')
+  end
+
+    it 'peep should be created and retrieved from database' do
+      expect(Peep.count).to eq(0)
```

Comment:
This should not be an expectation - it is a precondition and, arguably, not relevant